src/directory/directory.mjs

```diff
@@ -858,6 +858,11 @@ export const directory = {
             route: '/lib/logging/remote-configuration',
             filters: ['ios']
           },
+          {
+            title: 'Access Logs',
```

Comment:
Can we change the title to "Viewing logs"

Reply:
Title updated to "View Logs".  I'm keeping the verb tense the same as the other pages/titles.